src/SFA.Tl.Matching.Automation.Tests/Project/Framework/Helpers/PageInteractionHelper.cs

```diff
@@ -188,5 +235,133 @@ public static String GetText(By locator)
             IWebElement webElement = webDriver.FindElement(locator);
             return webElement.Text;
         }
+
+        public static String GetTextFromField(By locator)
+        {
+            IWebElement webElement = webDriver.FindElement(locator);
+            String text = webElement.GetAttribute("value");
+            return text;
+        }
+
+        public static void AssertText(String expectedText, String actualText)
+        {
+            Assert.AreEqual(expectedText, actualText);
+        }
```

Comment:
VerifyText method is already doing that with a custom exception, do we need this method again ?